lib/src/ast/sass/expression/variable.dart

```diff
@@ -22,6 +23,13 @@ class VariableExpression implements Expression {
 
   final FileSpan span;
 
+  FileSpan get nameSpan =>
+      namespace == null ? span : span.subspan(namespace!.length + 1);
```

Comment:
Rather than using `!`, assign `namespace` to a local variable so its non-nullability is inferred.